### PR TITLE
Rename jsp-3.0 feature to pages-3.0

### LINKED ISSUE #13521: Rename jsp-3.0 feature to pages-3.0
Related to discussions:
https://github.com/OpenLiberty/open-liberty/issues/13327
https://github.com/OpenLiberty/open-liberty/issues/13393

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #13521

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by ManasiGandhi** [comment]
@pnicolucci I want to review `dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/resources/l10n/com.ibm.websphere.appserver.pages-3.0.properties` but am not able to locate it in the files. Can you point me to the file for message review.

**Comment by tevans78** [comment]
@ManasiGandhi The file was only renamed so there is no content change that needs reviewing.

@pnicolucci I think you renamed it incorrectly ... You put `dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/resources/l10n/com.ibm.websphere.appserver.pages-3.0.properties` but I think it should have been `dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/resources/l10n/io.openliberty.pages-3.0.properties`

**Comment by ManasiGandhi** [comment]
@tevans78 Thanks for your reply.

**Comment by pnicolucci** [comment]
@tevans78 thanks for the review comment, I've pushed up the changes to this PR.

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by skasund** [comment]
L2 message review not required

**Comment by pnicolucci** [comment]
#build

**Comment by dmuelle** [comment]
Marking this as ID review not needed since it's just a filename change. If that's incorrect just let me know.

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Review by tevans78 — APPROVED** [review]

File: dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature [hunk 1]
```diff
@@ -29,10 +29,10 @@ IBM-API-Package: jakarta.servlet.jsp;  type="spec", \
  org.apache.el;  type="internal", \
  org.apache.el.lang;  type="internal", \
  org.apache.jasper.el;  type="internal"
-IBM-ShortName: jsp-3.0
+IBM-ShortName: pages-3.0
 IBM-SPI-Package: com.ibm.wsspi.jsp.taglib.config
```
Comment (cbridgha) [inline dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature:33]: Thinking you should also put a WLP-AlsoKnownAs: jsp-3.0

**Review by cbridgha — CHANGES_REQUESTED** [review]
looks good - just the suggestion to add the "known as" directive to help with error conditions

**Review by isaacrivriv — APPROVED** [review]
Looks good to me. Ill refer to Chuck on the feature file changes

File: dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature (hunk 1 above)
Comment (pnicolucci) [reply dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature:33]: actually I have an issue opened to do that: https://github.com/OpenLiberty/open-liberty/issues/14317

I'll work that after I get this initial merge done.

**Review by cbridgha — APPROVED** [review]

**Comment by pnicolucci** [comment]
#run-libby-bot

**Review by volosied — COMMENTED** [review]
I agree with the others. Thanks for working on this. :) 
All the `tested.features` must have been very annoying!

**Review by volosied — APPROVED** [review]

**Pull request merged**